src/modules/organizations/containers/links-container.jsx

```diff
@@ -45,31 +45,35 @@ class LinksContainer extends React.Component {
       <div>
         <h5>Links</h5>
         <div>
-          <span className="form__label" htmlFor="external urls">
+          <label className="form form__label" htmlFor="external urls">
             External Links
-            <ExternalLinksEditor
-              id="external"
-              name="external"
-              onChange={this.handleUrlsChange}
-              urls={this.state.urls}
-            />
-          </span>
+            <div className="form__input--top-margin">
+              <ExternalLinksEditor
+                id="external"
+                name="external"
+                onChange={this.handleUrlsChange}
+                urls={this.state.urls}
+              />
+            </div>
+          </label>
           <small className="form__help">
             Adding an external link will make it appear as a new tab alongside
             the about, classify, talk, and collect tabs.
           </small>
         </div>
         <br />
         <div className="form__fieldset">
-          <span className="form__label">
+          <label className="form form__label">
             Social Links
-            <SocialLinksEditor
-              id="social"
-              name="social"
-              onChange={this.handleUrlsChange}
-              urls={this.state.urls}
-            />
-          </span>
+            <div className="form__input--top-margin">
+              <SocialLinksEditor
+                id="social"
```

Comment:
We should add a `className` prop to this component, if it doesn't have one already, to avoid the wrapper `div`.